examples/interactivity/feature-click-style.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Feature click | CARTO</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8">
+    <!-- Include CARTO GL JS -->
+    <script src="../../dist/carto-gl.js"></script>
+    <!-- Include Mapbox GL JS -->
+    <script src="../../vendor/mapbox-gl-dev.js"></script>
+    <!-- Include Mapbox GL CSS -->
+    <link href="../../vendor/mapbox-gl-dev.css" rel="stylesheet" />
+    <style>
+        * {
+            margin: 0;
+            padding: 0;
+            box-sizing: border-box;
+        }
+
+        #map {
+            position: absolute;
+            height: 100%;
+            width: 100%;
+            z-index: 0;
+        }
+
+        #controls {
+            position: absolute;
+            top: 12px;
+            right: 12px;
+            width: 200px;
+            z-index: 2;
+        }
+
+        #controls h2 {
+            font: 300 12px/16px 'Open Sans';
+            background: rgba(0, 0, 0, 0.64);
+            border-radius: 4px;
+            padding: 8px 12px;
+            color: #fff;
+        }
+
+        .container {
+            padding: 8px 12px;
+            background: white;
+            box-shadow: 0 0 16px rgba(0, 0, 0, 0.12);
+            border-radius: 4px;
+            color: #2E3C43;
+            margin-top: 12px;
+        }
+
+        #content h3 {
+            font: 400 16px/22px 'Open Sans';
+            margin-bottom: 4px;
+        }
+
+        #content p {
+            font: 400 12px/16px 'Open Sans';
+        }
+    </style>
+</head>
+
+<body>
+    <div id="map"></div>
+    <div id="controls">
+        <h2></h2>
+        <div id="content">
+        </div>
+    </div>
+    <script>
+        const map = new mapboxgl.Map({
+            container: 'map',
+            style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+            center: [0, 30],
+            zoom: 2
+        });
+
+        carto.setDefaultAuth({
+            user: 'cartogl',
+            apiKey: 'YOUR_API_KEY'
+        });
+
+        const source = new carto.source.Dataset('ne_adm0_europe');
+        const style = new carto.Style(`
+      color: ramp($admin, bold)
+    `);
+        const layer = new carto.Layer('layer', source, style);
+
+        layer.on('loaded', () => {
+            updateTitle('Click on the countries');
+
+            const interactivity = new carto.Interactivity([layer]);
```

Comment:
this could be moved outside the on('loaded'

Reply:
Totally